Muehle_Utilities.ipynb

```diff
@@ -229,49 +261,6 @@
     "            result.append(board)\n",
     "    return result"
    ]
-  },
-  {
-   "source": [
-    "Die Funkiton `to_tupel()` wandelt einen übergebenen Spielstatus in Tupelform um."
-   ],
-   "cell_type": "markdown",
-   "metadata": {}
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "def to_tupel(state):\n",
-    "    return ((state[0][0], state[0][1]), (\n",
-    "                (state[1][0][0],state[1][0][1],state[1][0][2],state[1][0][3],state[1][0][4],state[1][0][5],state[1][0][6],state[1][0][7]),\n",
-    "                (state[1][1][0],state[1][1][1],state[1][1][2],state[1][1][3],state[1][1][4],state[1][1][5],state[1][1][6],state[1][1][7]),\n",
-    "                (state[1][2][0],state[1][2][1],state[1][2][2],state[1][2][3],state[1][2][4],state[1][2][5],state[1][2][6],state[1][2][7])\n",
-    "            ))"
-   ]
-  },
-  {
```

Comment:
hast du die Funktion irgendwo wieder eingesetzt? Ansonsten funktoiniert AB/minimax nicht. Gitl auch für to_list.

Reply:
=> to_tupel is not defined

Reply:
Jap, schau mal weiter oben. Ich hab die Funktionen stark gekürzt. Und es heißt to_tupLE im englischen, dementsprechend habe ich das umbenannt

Reply:
Im Algo-Notebook war das bei mir aber noch nicht umbenannt

Reply:
Ah stimmt, anscheinend hab ich die Datei nicht richtig gespeichert. Jetzt sollte es gleich passen.